blazor/chart/chart-types/bar.md

```diff
@@ -53,7 +53,7 @@ N> Refer to our [Blazor Bar Charts](https://www.syncfusion.com/blazor-components
 
 ## Bar space and width
 
-The [ColumnSpacing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_ColumnSpacing) and [ColumnWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_ColumnWidth) properties are used to customize the space between bars.
+The [ColumnSpacing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_ColumnSpacing) and [ColumnWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_ColumnWidth) properties are used to customize the space between bars. The default value of ColumnSpacing is 0, and the default value of ColumnWidth is 0.7. Both properties range from 0 to 1.
```

Comment:
Link the API reference link like previous description for `ColumnWidth` and `ColumnSpacing`. maintain uniformity.